app/models/post.rb

```diff
@@ -0,0 +1,17 @@
+class Post < ActiveRecord::Base
+  belongs_to :author, class_name: 'User', foreign_key: :user_id
+  has_many :likes
+  has_many :comments
+
+  def five_last_comments
+    comments.order('created_at Desc').limit(5)
+  end
+
+  after_save :update_posts_counter
+
+  private
+
+  def update_posts_counter
+    author.increment!(:posts_counter)
+  end
```

Comment:
- [ ] I got an error when running the seed. This is because here on line 15 you are writing `posts_counter` while in your schema you have `PostsCounter`. Kindly fix this

Reply:
This has been resolved.